core/src/main/resources/jenkins/model/Messages_tr.properties

```diff
@@ -20,20 +20,51 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-Hudson.BadPortNumber=Yanlış Port Numarası {0}
-Hudson.Computer.Caption=Master
-Hudson.Computer.DisplayName=master
-Hudson.ControlCodeNotAllowed=Kontrol koduna izin verilmiyor: {0}
+Hudson.BadPortNumber={0} geçersiz port numarası
+Hudson.Computer.Caption=Yerleşik Node
+Hudson.Computer.DisplayName=Yerleşik
+Hudson.Computer.IncorrectNumberOfExecutors="Executor sayısı" alanı geçersiz. Sıfır veya pozitif bir sayı olmalıdır.
+Hudson.ControlCodeNotAllowed=Kontrol kodu kullanılamaz: {0}
 Hudson.DisplayName=Jenkins
-Hudson.JobAlreadyExists=''{0}'' isminde bir iş zaten mevcut
-Hudson.NoJavaInPath=java, PATH içerisinde değil. <a href=''{0}/configure''>JDK konfigürasyonunu</a> düzeltmeniz gerekebilir?
-Hudson.NoName=İsim belirtilmedi
+Hudson.JobAlreadyExists=''{0}'' adında bir job zaten mevcut
+Hudson.NoJavaInPath=java PATH içinde bulunamadı. Belki <a href="{0}/configure">JDK yapılandırması</a> yapmanız gerekiyor?
+Hudson.NoName=İsim belirtilmemiş
+Hudson.NodeBeingRemoved=Node kaldırılıyor
 Hudson.UnsafeChar=''{0}'' güvenli olmayan bir karakter
-Hudson.ViewName=Hepsi
-ParameterizedJobMixIn.build_with_parameters=Parametrelerle Yapılandır
```

Comment:
Another instance where you change the translation of the word 'build' to a different value. This word is heavily used across Jenkins. Changing it for a few instances will cause huge confusion.